browser-test/src/applicant/application_version_fast_forward.test.ts

```diff
@@ -984,6 +984,407 @@ test.describe('Application Version Fast-Forward Flow', () => {
     await applicantActor.closeBrowserContext()
     await programAdminActor.closeBrowserContext()
   })
+
+  test('all major steps - fast forward flag enabled - disabled visibility enabled', async ({
```

Comment:
**question:** Just confirming everything added here is brought over from the existing test, right? If not, would maybe be helpful to note which the new pieces are.

Reply:
It's the same except the end when it checks the page redirected. Now that it's starting to grow I going to go back later and find a way to reduce the duplication.

Reply:
Sounds good, thanks!!